in-toto-spec.md

```diff
@@ -935,7 +935,8 @@ information about the step performed. Byproducts are not verified by in-toto’s
 default verification routine. However, the information gathered can be used for
 further scrutiny during an inspection step. At a minimum, the byproducts
 dictionary should have standard output (stdout), standard error (stderr) and
-return value (return-value), even if no values are filled in.
+return value (return-value), even if no values are filled in. The return value
+is stored as integer value.
```

Comment:
Maybe we can make this a *should be stored*, given that we claim `byproducts` is an opaque dictionary.